camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -1,344 +1,349 @@
+                          <FormView :formioUrl="formioUrl"/>
+                        </div>
+                      </b-overlay>
+                    </div>
+                  </b-tab>
+                  <b-tab title="History">
+                    <TaskHistory
+                      :taskHistoryList="taskHistoryList"
+                      :applicationId="applicationId"
+                    />
+                  </b-tab>
+                  <b-tab
+                    class="cft-diagram-container"
+                    id="diagramContainer"
+                    title="Diagram"
                   >
-                    <span id="claimAssignee">
-                      <i class="fa fa-user" /> Claim
-                    </span>
-                  </div>
-                </span>
-              </b-col>
-            </b-row>
-            <div class="height-100">
-              <b-tabs pills class="height-100" content-class="mt-3">
-                <b-tab title="Form" active>
-                  <div v-if="showForm" class="ml-4 mr-4 form-tab-conatiner">
-                    <b-overlay
-                      :show="task.assignee !== userName"
-                      variant="light"
-                      opacity="0.75"
-                      spinner-type="none"
-                      aria-busy="true"
-                    >
-                      <div v-if="task.assignee === userName">
-                        <FormEdit
-                          :formioUrl="formioUrl"
-                          @onformsubmit="onFormSubmitCallback"
-                          @oncustomevent="oncustomEventCallback"
-                        />
-                      </div>
-                      <div v-else>
-                        <FormView :formioUrl="formioUrl"/>
-                      </div>
-                    </b-overlay>
-                  </div>
-                </b-tab>
-                <b-tab title="History">
-                  <TaskHistory
-                    :taskHistoryList="taskHistoryList"
-                    :applicationId="applicationId"
-                  />
-                </b-tab>
-                <b-tab
-                  class="cft-diagram-container"
-                  id="diagramContainer"
-                  title="Diagram"
-                >
-                  <div class="diagram-full-screen" id="canvas"></div>
-                </b-tab>
-              </b-tabs>
+                    <div class="diagram-full-screen" id="canvas"></div>
+                  </b-tab>
+                </b-tabs>
+              </div>
             </div>
           </div>
+        </b-col>
+        <b-col v-else>
+          <b-row class="cft-not-selected mt-2 ml-1 row">
+            <i
+              class="fa fa-exclamation-circle-fill"
+              variant="secondary"
+              scale="1"
+            ></i>
+            <p>Select a task in the list.</p>
+          </b-row>
+        </b-col>
+      </b-row>
+    </b-container>
+    <b-container v-else>
```

Comment:
It should have been <div v-else here>. Layout is affected for Unauthorized user.

![image](https://user-images.githubusercontent.com/70306694/135948776-ca64b3d5-6cd8-4262-ae84-838ff2802c4e.png)

Reply:
This was caused by missing fluid class. It's okay to not change this.